zigpy/zcl/clusters/security.py

```diff
@@ -153,10 +153,7 @@ def handle_cluster_request(
             and self.is_server
             and not hdr.frame_control.disable_default_response
         ):
-            hdr.frame_control = hdr.frame_control.replace(
-                direction=Direction.Client_to_Server
-            )  # this is a client -> server cmd
```

Comment:
Correct me if I'm wrong, but `hdr.frame_control` doesn't affect `send_default_rsp` in any way. There could be some Python magic that I'm missing though

Reply:
I think the intent was originally to flip the direction in the header and send a reply *using* that header, since without this the header for the reply would be wrong.